### DISCUSSION TITLE
How to upgrade home assistant addon edge

### DISCUSSION THREAD

**Discussion by maframan** [Koenkk/zigbee2mqtt#14729] [General]
Hi, I need to understand procedure for upgrade edge addon without loose my network configuration

**Comment by Koenkk** [comment]
You can do this by uninstalling and installing the addon. If you want to be sure update the `/share/zigbee2mqtt` (or `/config/zigbee2mqtt` dir)

**Reply by maframan** [reply]
Thanks a lot!
